messages/german.json

```diff
@@ -0,0 +1,1393 @@
+        "Die Dorfbewohner haben es schwer mit dem Schmerz des Todes von {0:@} und lynchten wiederwillig {1!role:article} {1!role:bold}.",
+        "Konform mit dem Willen des Dorfes, erwiesen sich die Galgen als wirksam bei der Tötung von {0:@}, {1!role:article} {1!role:bold}.",
+        "In einem Anfall von Hysterie lynchen die Dorfbewohner {0:@}, und töten {1!role:article} {1!role:bold}.",
+        "Während sie auf ihre Hinrichtung warten, fällt ein Baum auf {0:@}, dies tötet {1!role:article} {1!role:bold}.",
+        "Die Dorfbewohner fesseln {0:@}. Und werfen {0:@} in einen See. Es scheint so das {1!role:article} {1!role:bold} nicht gut schwimmen kann",
+        "Glaubt man ihrem Nachbarn und Freund das {0:f@} gefährlich zu sein scheint, so legt die Masse der Dorfbewohner den {1!role:bold} {0:@} um.",
+        "Unter viel Lärm lynchen die Pechgabel-lagerischen Dorfbewohner {0:@}, was sich als... {1!role:article} {1!role:bold} entpuppte.",
+        "Trotz Protesten schleppt der Mob sein Opfer an den hängenden Baum. {0:@} erliegt dem Willen der Horde und wird gehängt. Die Dorfbewohner haben {1!role:article} {1!role:bold} getötet .",
+        "Resigniert mit dem Unvermeidlichen wird {0:@} zum Galgen geführt. Sobald das Zucken aufhörte, wurde entdeckt, dass das Dorf {1!role:article} {1!role:bold} gelyncht hat.",
+        "Resigniert mit dem Unvermeidlichen wird {0:@} zum Galgen geführt. Danach inspiziert das Dorf die Leiche und findet {1!role:article} {1!role:bold}.",
+        "Resigniert mit dem Unvermeidlichen wird {0:@} zum Galgen geführt. Leider reist das Seil, und {1!role:article} {1!role:bold} schlägt auf den Boden auf und stirbt.",
+        "Bevor das Seil gezogen wird, {0:@}, {1!role:article} {1!role:bold}, zieht den Stift an einer Granate. Sie zögert, und explodiert und tötet {0:@}.",
+        "Bevor das Seil gezogen wird wirft {0:@} {1!role:article} {1!role:bold}, eine Granate auf Menge doch die Granate explodierte zu früh.",
+        "Die Dorfbewohner ziehen {0:@} an den Rand einer Klippe. Bevor sie protestieren können, {1!role:article} {1!role:bold} rutscht und fällt in den Abgrund.",
+        "Mit einem schaurigen Kriegsschrei ziehen die Dorfbewohner {0:@} an den Rand einer Klippe und werfen {0:@} hinab. Die Felswand ist nun mit den Rester von {0:@} {1!role:article}  {1!role:bold} bedeckt.",
+        "{0:@} wird auf ein Rudel hungriger Tiere geworfen. das {1!role:bold} leben hält nicht lange.",
+        " {0:@} into a volcano. They discover the melting point of {1!role:article} {1!role:bold} is less than that of lava.",
+        "Für die WISSENSCHAFT, die Dorfbewohner fesseln {0:@} an einen Elektrischen Stuhl und Schalten den Strom ein. Der Verbrannte Körper identifizierte sich dann als {1!role:article} {1!role:bold}.",
+        "Die wütenden Dorfbewohner packen {0:@} an den Beinen und Armen, und werfen {0:@} in ein großes Feuer. Es scheint, dass {1!role:article} {1!role:bold}'s Körper schließlich nicht so gut brennt",
```

Comment:
would remove the comma before the "und". Possessive form is going to be problematic. Would remove "schließlich". Missing final period